src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -1305,7 +1305,10 @@ private static bool IsProxyForCmdlet(Dictionary<string, ParameterMetadata> param
             // we are not sending CmdletBinding/DefaultParameterSet over the wire anymore
             // we need to infer IsProxyForCmdlet from presence of all common parameters
 
-            foreach (string commonParameterName in Cmdlet.CommonParameters)
+            // need to exclude `ProgressAction` which may not exist for downlevel platforms
+            var commonParameters = Cmdlet.CommonParameters;
+            commonParameters.Remove("ProgressAction");
```

Comment:
1. The calling to `Remove` here removes `ProgressAction` from the static hash set `s_commonParameters`. It will likely cause problems with other places that depends on `s_commonParameters`.
2. We should ignore `ProgressAction` only if the server side is a down-level version that doesn't support this common parameter. You may be able to use the same check code from here:
https://github.com/PowerShell/PowerShell/blob/963bf1e0ade35e0d954196d0b7a61da30e3d5bd1/src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs#L1576-L1581